charts/chronicle/templates/opa-init.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.opa.enabled }}
+{{$stlServiceName := include "lib.call-nested" (list . "sawtooth" "common.names.fullname")}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-opa-init
+  labels: {{ include "chronicle.labels" . | nindent 4 }}
+    component: opa-init
+spec:
+  backoffLimit: 5
+  template:
+    metadata:
+      labels: {{- include "chronicle.labels" . | nindent 8 }}
+        component: opa-init
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ include "lib.serviceAccountName" . }}
+      automountServiceAccountToken: true
+      affinity: {{ include "lib.safeToYaml" .Values.affinity | nindent 8 }}
```

Comment:
don't use affinity here.  Jobs shouldn't matter where they are run

Reply:
You'll see in subsequent changes I've made `opa-init` an initContainer within the `chronicle` pod.